src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ProbationAreaRepository.java

```diff
@@ -6,11 +6,15 @@
 import org.springframework.data.repository.query.Param;
 import uk.gov.justice.digital.delius.jpa.standard.entity.ProbationArea;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface ProbationAreaRepository extends JpaRepository<ProbationArea, Long>, JpaSpecificationExecutor<ProbationArea> {
     Optional<ProbationArea> findByCode(String code);
 
     @Query("select pa from ProbationArea pa, RInstitution institution where pa.institution = institution and institution.nomisCdeCode = :nomisCdeCode")
     Optional<ProbationArea> findByInstitutionByNomsCDECode(@Param("nomisCdeCode") String nomsPrisonInstitutionCode);
+
+    @Query("select pa from ProbationArea pa, RInstitution institution where pa.institution = institution and institution.nomisCdeCode is not null and institution.nomisCdeCode <> 'OUT'")
```

Comment:
get all but exclude the special "OUT" prison, that obviously isn't a real prison